src/styles.scss

```diff
@@ -49,7 +49,6 @@ button, textarea, input, select, a {
 
 .ql-editor {
   width: 100%;
-  min-height: 400px;
```

Comment:
This was causing campaign view on mobile to have too much empty space if the description was very short. Checked for other usages, everything still looks fine with this change. There is a class override on admin screen which uses `::ng-deep` selector - this can be removed now (I didn't do it now to avoid conflicts).
Before:
![Screenshot 2021-09-22 at 15 52 48](https://user-images.githubusercontent.com/12549625/134381880-7e0b5ad2-4de0-4b3f-b2cc-88ffabd0d0af.png)
After:
![Screenshot 2021-09-22 at 15 53 25](https://user-images.githubusercontent.com/12549625/134381884-81dcb147-062e-4f15-9323-7c4495d69236.png)